scripts/auto-rebase/rebase.sh

```diff
@@ -627,6 +627,7 @@ rebase_to() {
     if [[ -n "$(git status -s pkg/release)" ]]; then
         title "## Committing changes to pkg/release"
         git add pkg/release
+        git add packaging/crio.conf.d
```

Comment:
@fzdarsky PTAL

Reply:
makes sense since we are also updating those